test/e2e/pages/hostingPage.js

```diff
@@ -11,11 +11,11 @@ export default {
     descriptionInput: {
       selector: '[data-e2e="hosting-dialog-description-input"]'
     },
-    domainsInput: {
-      selector: '[data-e2e="hosting-dialog-domains-input"]'
+    newDomainInput: {
+      selector: '[name="newDomain"]'
```

Comment:
Should we really change `data-e2e` attributes to names? If someone refactors it in future it is not so obvious that something targets it, like tests.